test/functional/repository/basic-methods/repository-basic-methods.ts

```diff
@@ -440,7 +440,7 @@ describe("repository > basic methods", () => {
         it("should first create then update an entity", () =>
             Promise.all(
                 connections.map(async (connection) => {
-                    if (connection.driver.supportedUpsertType == null) return
+                    if (!connection.driver.supportedUpsertTypes) return
```

Comment:
@AlexMesser best if we use empty arrays and remove undefined state from the `supportedUpsertTypes`. Less types - less problems.